madara/crates/tests/src/lib.rs

```diff
@@ -131,9 +131,15 @@ impl Drop for MadaraCmd {
 
         // Ensure process cleanup
         let _ = child.wait();
+        std::thread::sleep(Duration::from_millis(500));
```

Comment:
I think a `sleep()` inside a `Drop` impl is probably asking for trouble, although I see that we did it just above this as well...

It is inside a `tests` crate, though... :thinking: